base/shell/explorer/taskswnd.cpp

```diff
@@ -690,6 +1134,16 @@ class CTaskSwitchWnd :
             return UpdateTaskItemButton(TaskItem);
         }
 
+        /*if (m_IsGroupingEnabled &&
```

Comment:
What was the original intended purpose of this commented code?

Reply:
I think i moved this behaviour to the CollapseOrExpand or i modified it in a way it's not needed to check for it